sensor_msgs/JointState joint_state
sensor_msgs/JointState[] configuration
MinibotPose pose

uint32 id
uint32 uid
string name
